README.md

```diff
@@ -61,11 +61,20 @@ python train.py --algo algo_name --env env_id
 ```
 You can use `-P` (`--progress`) option to display a progress bar.
 
-Using a custom yaml file (which contains a `env_id` entry):
+Using a custom config file when it is a yaml file with a which contains a `env_id` entry:
 ```
-python train.py --algo algo_name --env env_id --yaml-file my_yaml.yml
+python train.py --algo algo_name --env env_id --conf-file my_yaml.yml
 ```
 
+You can also use a python file that contains a dictionary called `hyperparams` with an entry for each `env_id`.
+(see `hyperparams/ppo_python_config_example.py` for an example)
+```
+python train.py --algo ppo --env MountainCarContinuous-v0 --conf-file hyperparams.ppo_python_config_example
```

Comment:
maybe add a note that `--conf-file hyperparams/ppo_python_config_example.py` does work too (more natural for python developpers)